docs/advanced-techniques.md

```diff
@@ -84,79 +84,6 @@ class Application extends React.Component {
 
 This way, the document head is specified in an isomorphic format so it can be rendered on the server and dynamically modified on the client. Use this approach to make your reSolve applications SEO-friendly.
 
-## Process Managers (Sagas)
-
-Process Managers (or Sagas) are used to run arbitrary service code in response to events or on schedule. Generally, this is where you define logic that deal with side effects: you can emit new events and communicate with the outside world in any way (e.g., query databases, send emails, etc.). You can view a Saga as a scripted virtual user.
-
-The code below demonstrates a Saga that handles events:
-
-```js
-const eventHandlers = {
-  UserCreationRequested: async (event, { resolve }) => {
-    const { aggregateId } = event
-    const createdUser = await resolve.executeQuery({
-      modelName: 'default',
-      resolverName: 'createdUser',
-      resolverArgs: { id: aggregateId }
-    })
-
-    if (!createdUser) {
-      return
-    }
-
-    const users = await resolve.executeQuery({
-      modelName: 'default',
-      resolverName: 'users',
-      resolverArgs: { id: aggregateId }
-    })
-
-    const userWithSameEmail = users.find(
-      user => user.email === createdUser.email
-    )
-
-    await resolve.executeCommand({
-      type: userWithSameEmail ? 'rejectUserCreation' : 'confirmUserCreation',
-      aggregateName: 'user',
-      payload: { createdUser },
-      aggregateId
-    })
-  }
-}
-
-export default eventHandlers
-```
-
-For a scheduled Saga, tasks are specified in the cron format. The code below demonstrates a Saga that emits commands on schedule:
-
-```js
-const outdatedPeriod = 1000 * 60 * 10
-
-const cronHandlers = {
-  '0 */10 * * * *': async ({ resolve }) => {
-    const users = await resolve.executeQuery({
-      modelName: 'default',
-      resolverName: 'users'
-    })
-
-    const now = Date.now()
-
-    users.forEach(user => {
-      if (user.timestamp + outdatedPeriod < now) {
-        resolve.executeCommand({
-          type: 'deleteOutdatedUser',
-          aggregateName: 'user',
-          aggregateId: user.id
-        })
-      }
-    })
-  }
-}
-
-export default cronHandlers
-```
-
-For the full code, refer to the **With Saga** example project.
-
 ## Adapters
 
 ReSolve uses the **adapter** mechanism to provide an abstraction layer above APIs used by its subsystems. For instance, adapters are used to define how a reSolve application stores its data. They abstract away all direct interactions with the underlying storage, allowing reSolve to provide a unified data management API.
```

Comment:
Can we change "APIs" to "API"?

Also change "They abstract away all direct interactions with..." to "They abstract all direct interaction with..."